src/taxpasta/domain/model/wide_observation_table.py

```diff
@@ -29,9 +31,16 @@ class WideObservationTable(pa.DataFrameModel):
     """Define the observation matrix."""
 
     taxonomy_id: Series[pd.CategoricalDtype] = pa.Field()
+    name: Optional[Series[pd.CategoricalDtype]] = pa.Field()
+    rank: Optional[Series[pd.CategoricalDtype]] = pa.Field()
+    lineage: Optional[Series[pd.CategoricalDtype]] = pa.Field()
+    id_lineage: Optional[Series[pd.CategoricalDtype]] = pa.Field()
+    rank_lineage: Optional[Series[pd.CategoricalDtype]] = pa.Field()
     # This field uses a regex to match all columns that are not `taxonomy_id`.
     any_samples: Series[np.int64] = pa.Field(
-        ge=0, alias="^(?!taxonomy_id$).*", regex=True
+        ge=0,
+        alias="^(?!(taxonomy_id|name|rank|lineage|id_lineage|rank_lineage)$).*",
+        regex=True,
```

Comment:
The comments says that this field uses a regex to match all columns that are not `taxonomy_id`. But if I understand correct the regex, it does not match the comment above.

Reply:
True, I need to update the comment. Thanks!